### PR TITLE
Features/02 dashboard eventpanel abdi

### PR DESCRIPTION
**Pull request by abwork**

### CODE REVIEW

**Comment by trungh13** [comment]
Thanks that I'm missing these in public/index.html
<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">   
But please don't change the boilerplate of index.html.

File: src/index.js [hunk 1]
```diff
@@ -2,16 +2,26 @@ import 'materialize-css/dist/css/materialize.min.css';
...
 import { BrowserRouter as Router } from 'react-router-dom';
+import reducers from './store/reducers';
+import thunk from 'redux-thunk';
+
 
 import App from './App';
 
-import reducers from './store/reducers';
+//middleware and initialstates
+const initialState = {};
```
Comment (rangigo) [inline src/index.js]: Init states should be declared in the reducers

File: src/index.js [hunk 2]
```diff
@@ -2,16 +2,26 @@ import 'materialize-css/dist/css/materialize.min.css';
...
+const middleware = [thunk];
```
Comment (rangigo) [inline src/index.js]: We probably do not need multiple middlewares for redux. Only thunk is needed

File: src/index.js [hunk 3]
```diff
@@ -2,16 +2,26 @@ import 'materialize-css/dist/css/materialize.min.css';
...
+
 /*eslint-disable */
 const store = createStore(
   reducers,
-  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__(),
+  initialState,
+  compose(
```
Comment (rangigo) [inline src/index.js]: compose can be a separate variable
E.g:  
```
const composeEnhancers =
  process.env.NODE_ENV === 'development'
    ? window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
    : null || compose;

const store = createStore(rootReducer, composeEnhancers(applyMiddleware(thunk)));
```

Comment (abwork) [reply src/index.js]: Resolved. I had to disable Eslint to ignore "no-underscore-dangle" issue so we can peacefully use redux devtools

File: src/index.js (hunk 2 above)
Comment (abwork) [reply src/index.js]: I agree. No more multiple middlewares.

File: src/index.js (hunk 1 above)
Comment (abwork) [reply src/index.js]: Init states is now declared in the reducer. More clean now :)

File: src/index.js [hunk 4]
```diff
@@ -2,19 +2,22 @@ import 'materialize-css/dist/css/materialize.min.css';
...
-import reducers from './store/reducers';
-/*eslint-disable */
-const store = createStore(
-  reducers,
-  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__(),
-);
+/* eslint-disable no-underscore-dangle */
+const composeEnhancers = process.env.NODE_ENV === 'development'
+  ? window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
+  : null || compose;
```
Comment (Nizar-Rahme) [inline src/index.js]: No need here for `null`

**Pull request closed without merging**